scripts/jenkins/Jenkinsfile

```diff
@@ -415,6 +421,52 @@ kill `pidof dbus-daemon`
     }
 }
 
+/* Generate Stages that build Debian packages
+ */
+def generate_package_build_stages() {
+    def tasks = [:]
+    tasks << build_package_debian_stretch()
+    return tasks
+}
+
+def build_package_debian_stretch() {
```

Comment:
why are some functions camel case and others use `_` ? (Not to be changed in this PR, though)

Reply:
again i am not even sure if i followed it consequently hence it is not final to stay that way yet.

Reply:
Okay, as said this does not have to be resolved within this PR.